lib/commands/publish/publishChangedPackages.js

```diff
@@ -127,7 +111,7 @@ module.exports = function publishChangedPackages(
 
     npmPublishSemiAtomic(changedPackages, packagesLoc, versions, function (err) {
       if (err) return onError(err);
-      gitPushWithTags();
+      gitUtils.pushWithTags();
```

Comment:
Oh, this actually conflicts with #54 also. :grimacing: 

Happy to update again if this lands first.